llm-request-plz.el

```diff
@@ -152,7 +152,7 @@ the buffer is turned into JSON and passed to ON-SUCCESS."
             (when on-success-raw
               (funcall on-success-raw response))
             (when on-success
-              (funcall on-success (json-read-from-string response))))
+              (funcall on-success (plz-response-body response))))
```

Comment:
I noticed the `plz-response-body` is either `nil` or an empty string, depending on the media type.  It should be consistent; probably `nil` I think.

Reply:
Yes, I changed this recently. It should be nil now, but let me double check and add tests for it.

Reply:
I doubled checked. This is now always nil if the body is handled in a streaming way.

Reply:
Thanks!